bmi_visualization.py

```
@@ -138,15 +140,17 @@ def _create_bmi_gauge(self, ax, bmi: float, group: str, lang: str):
-        ax.set_xlim(15, 40)
+        ax.set_xlim(ranges[0][0], ranges[-1][1])
         ax.set_ylim(-1, 1)
         ax.set_xlabel("BMI Value", fontsize=12)
         ax.set_yticks([])
         ax.grid(axis="x", alpha=0.3)
         ax.legend(loc="upper right")
         ax.set_title(f"Current BMI: {bmi}", fontsize=14, fontweight="bold")
 
-    def _create_guidance_chart(self, ax, bmi: float, age: int, gender: str, group: str, lang: str):
+    def _create_guidance_chart(
+        self, ax: "Axes", bmi: float, age: int, _gender: str, group: str, lang: str
```

Ruff (0.14.0)

Unused method argument: `age`

(ARG002)